nengo/spa/spa.py

```diff
@@ -88,7 +88,11 @@ def __setattr__(self, key, value):
 
     def get_module(self, name):
         """Return the module for the given name."""
-        return self._modules[name]
+        if name in self._modules:
+            return self._modules[name]
+        elif '_' in name:
+            module, name = name.rsplit('_', 1)
```

Comment:
Yup, that should be documented somewhere.  In fact, I really don't like the underscore convention here, but nothing else was coming to mind.